src/hotspot/os_cpu/bsd_x86/os_bsd_x86.cpp

```diff
@@ -678,29 +636,6 @@ JVM_handle_bsd_signal(int sig,
     return true;
   }
 
-  // signal-chaining
-  if (PosixSignals::chained_handler(sig, info, ucVoid)) {
-     return true;
-  }
-
-  if (!abort_if_unrecognized) {
-    // caller wants another chance, so give it to him
-    return false;
-  }
-
-  if (pc == NULL && uc != NULL) {
-    pc = os::Bsd::ucontext_get_pc(uc);
-  }
-
-  // unmask current signal
-  sigset_t newset;
-  sigemptyset(&newset);
-  sigaddset(&newset, sig);
-  sigprocmask(SIG_UNBLOCK, &newset, NULL);
```

Comment:
This is not being moved to javaSignalHandler_inner(), so why are we dropping this completely?

Reply:
I just saw the answer to this in the thread, nvm.